Zilon.Core/Zilon.Bot.Players.DevelopmentTests/AutoplayEngine.cs

```diff
@@ -0,0 +1,240 @@
+                var equipment = propFactory.CreateEquipment(equipmentScheme);
+                equipmentCarrier[slotIndex] = equipment;
+            }
+            catch (KeyNotFoundException)
+            {
+                Debug.WriteLine($"Не найден объект {equipmentSid}");
+            }
+        }
+
+        private static void AddEquipmentToActor(Inventory inventory, string equipmentSid,
+            ISchemeService schemeService, IPropFactory propFactory)
+        {
+            try
+            {
+                var equipmentScheme = schemeService.GetScheme<IPropScheme>(equipmentSid);
+                var equipment = propFactory.CreateEquipment(equipmentScheme);
+                inventory.Add(equipment);
+            }
+            catch (KeyNotFoundException)
+            {
+                Debug.WriteLine($"Не найден объект {equipmentSid}");
+            }
+        }
+
+        private static void AddResourceToActor(Inventory inventory, string resourceSid, int count,
+            ISchemeService schemeService, IPropFactory propFactory)
+        {
+            try
+            {
+                var resourceScheme = schemeService.GetScheme<IPropScheme>(resourceSid);
+                var resource = propFactory.CreateResource(resourceScheme, count);
+                inventory.Add(resource);
+            }
+            catch (KeyNotFoundException)
+            {
+                Debug.WriteLine($"Не найден объект {resourceSid}");
+            }
+        }
+
+
+        private async Task<IActor> CreateSectorAsync(IServiceProvider _globalServiceProvider)
+        {
+            if (_serviceScope != null)
+            {
+                _serviceScope.Dispose();
+                _serviceScope = null;
+            }
+
+            _serviceScope = _globalServiceProvider.CreateScope();
+
+            _startup.ConfigureAux(_serviceScope.ServiceProvider);
+
+            var schemeService = _globalServiceProvider.GetRequiredService<ISchemeService>();
+            var humanPlayer = _globalServiceProvider.GetRequiredService<HumanPlayer>();
+            var survivalRandomSource = _globalServiceProvider.GetRequiredService<ISurvivalRandomSource>();
+            var propFactory = _globalServiceProvider.GetRequiredService<IPropFactory>();
+            var scoreManager = _globalServiceProvider.GetRequiredService<IScoreManager>();
+
+            var gameLoop = _serviceScope.ServiceProvider.GetRequiredService<IGameLoop>();
+            var sectorManager = _serviceScope.ServiceProvider.GetRequiredService<ISectorManager>();
+            var botActorTaskSource = _serviceScope.ServiceProvider.GetRequiredService<HumanBotActorTaskSource>();
+            var actorManager = _serviceScope.ServiceProvider.GetRequiredService<IActorManager>();
+            var monsterActorTaskSource = _serviceScope.ServiceProvider.GetRequiredService<MonsterBotActorTaskSource>();
+
+            await sectorManager.CreateSectorAsync().ConfigureAwait(false);
+
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5014856)